plugins/sinks/columbus/columbus.go

```diff
@@ -1,20 +1,26 @@
 package columbus
 
 type Record struct {
```

Comment:
we can rename this to be more clear e.g. `RequestPayload`. wdyt?

Reply:
`Record` specify more about what actually model is about as it is known to be a payload only, while `RequestPayload` is a general term for payload request for any api . So should we change it ?

Reply:
yes that worked before, but now Columbus does not have Record and only has Asset.